composer.json

```diff
@@ -50,7 +50,7 @@
         "ramsey/uuid-doctrine": "^1.5",
         "react/http": "^1.0",
         "sensio/framework-extra-bundle": "^5.5",
-        "sentry/sentry-symfony": "^3.4",
+        "sentry/sentry-symfony": "^4.0",
```

Comment:
AFAIK this update is not just as straightforward as presented (no changes to [the configuration](https://github.com/repman-io/repman/blob/a628a0a3969427dec8d444c5459bc2ec618e08c5/config/packages/prod/sentry.yaml)).

See `Removed the sentry.options.excluded_exceptions configuration option.`

* https://github.com/getsentry/sentry-symfony/blob/28ebfb37940e4f41e4fac9f0400e9e9439d1e4cc/UPGRADE-4.0.md

Reply:
Ah good catch, I'll fix this tomorrow by using the intergration instead!

Reply:
@marmichalski I've pushed support for this, can you do another review round?